sdk/vision/azure-ai-vision-imageanalysis/README.md

```diff
@@ -0,0 +1,340 @@
+
+Use the Image Analysis client library to:
+* Authenticate against the service
+* Set what features you would like to extract
+* Upload an image for analysis, or send an image URL
+* Get the analysis result
+
+[Product documentation][image_analysis_overview] 
+| [Samples][samples]
+| [Vision Studio][vision_studio]
+| [API reference documentation](https://learn.microsoft.com/java/api/com.azure.ai.vision.imageanalysis)
+| [Maven Package](https://mvnrepository.com/artifact/com.azure/azure-ai-vision-imageanalysis)
+| [SDK source code][sdk_source_code]
+
+## Getting started
+
+### Prerequisites
+
+*  [Java Development Kit (JDK)](https://learn.microsoft.com/azure/developer/java/fundamentals/java-jdk-install) with version 8 or above.
+* An [Azure subscription](https://azure.microsoft.com/free).
+* A [Computer Vision resource](https://portal.azure.com/#create/Microsoft.CognitiveServicesComputerVision) in your Azure subscription.
+  * You will need the key and endpoint from this resource to authenticate against the service.
+  * You can use the free pricing tier (`F0`) to try the service, and upgrade later to a paid tier for production.
+  * Note that in order to run Image Analysis with the `Caption` or `Dense Captions` features, the Azure resource needs to be from one of the following GPU-supported regions: `East US`, `France Central`, `Korea Central`, `North Europe`, `Southeast Asia`, `West Europe`, or `West US`.
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-ai-vision-imageanalysis;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-ai-vision-imageanalysis</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Set environment variables
+
+To authenticate the `ImageAnalysisClient`, you will need the endpoint and key from your Azure Computer Vision resource in the [Azure Portal](https://portal.azure.com). The code snippet below assumes these values are stored in environment variables:
+
+* Set the environment variable `VISION_ENDPOINT` to the endpoint URL. It has the form `https://your-resource-name.cognitiveservices.azure.com`, where `your-resource-name` is your unique Azure Computer Vision resource name.
+
+* Set the environment variable `VISION_KEY` to the key. The key is a 32-character Hexadecimal number.
+
+### Create and authenticate the client
+
+Once you define the environment variables, this Java code will create and authenticate a synchronous `ImageAnalysisClient`:
+
+```java imports-for-create-client-snippet
+import com.azure.ai.vision.imageanalysis.ImageAnalysisClient;
+import com.azure.ai.vision.imageanalysis.ImageAnalysisClientBuilder;
+import com.azure.core.credential.KeyCredential;
+```
+```java create-client-snippet
+String endpoint = System.getenv("VISION_ENDPOINT");
+String key = System.getenv("VISION_KEY");
```

Comment:
nit, for sample purposes I'd recommend using `azure-core`'s `Configuration` which can load configurations from many sources (defaulted to environment variables and Java system properties)

Reply:
Hi @alzimmermsft, thank you for this comment! I looked at the Configuration class (I was not familiar with it). I tried it out. Looks like instead of 
```java
String endpoint = System.getenv("VISION_ENDPOINT");
String key = System.getenv("VISION_KEY");
```
I can use:
```java
import  com.azure.core.util.Configuration;
import  com.azure.core.util.ConfigurationBuilder;

Configuration configuration = new ConfigurationBuilder().build();
String endpoint = configuration.get("VISION_ENDPOINT");
String key = configuration.get("VISION_KEY");
```
That worked great. However, I want to keep the samples simple and not introduce any new dependencies that will require developers to learn a new class unrelated to the Image Analysis SDK. I see that using System.getenv() is very common in sample code, so I'd rather keep it as is if that's okay with you.

Reply:
I see no issue with leaving the sample using System.getenv(). `Configuration` should automatically be available as a transitive dependency from `azure-core`, but it does add some cognitive overhead as it's specific to the Azure SDKs and not a Java language concept.